plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/service/JavaLanguageExtensionServiceClient.java

```diff
@@ -259,4 +262,25 @@ private void onTimeout(RejectFunction reject) {
             new TimeoutException(
                 "Looks like the language server is taking to long to respond, please try again in sometime.")));
   }
+
+  public Promise<UsagesResponse> usages(TextDocumentPositionParams params) {
+    return Promises.create(
+        (resolve, reject) -> {
+          requestTransmitter
+              .newRequest()
+              .endpointId(WS_AGENT_JSON_RPC_ENDPOINT_ID)
+              .methodName("java/usages")
+              .paramsAsDto(params)
+              .sendAndReceiveResultAsDto(UsagesResponse.class, 10000)
```

Comment:
Maybe it would be better to move constant into `Constants` to reuse in `JavaLanguageServerExtensionService`?
Or, if it is the same as default request timeout, just use the constant?